application/job/job_fixtures_test.go

```diff
@@ -22,6 +27,27 @@ func createFixture(js *JobSuite, f interface{}) {
 	}
 }
 
+func CreateFixtures_TestOutdatedRequestHandler(js *JobSuite) RequestFixtures {
+	users := test.CreateUserFixtures(js.DB, 2).Users
+	requests := test.CreateRequestFixtures(js.DB, 2, false, users[0].ID, users[1].ID)
+	db := js.DB
+
+	requests[0].Status = models.RequestStatusAccepted
+	js.NoError(requests[0].Update(db), "error modifying request for test")
+
+	pastDate := nulls.NewTime(time.Now().Add(-1 * domain.DurationDay))
+	requests[1].NeededBefore = pastDate
+	// avoid validation error for value in the past
```

Comment:
FYI, you can use Pop without triggering validation. Just use `tx.Update` instead of `tx.ValidateAndUpdate`.